### PR TITLE
[TEST][WIP] Update CoreDNS config to 1.5

### PR DESCRIPTION
**Pull request by aojea**
Add the ready plugin, otherwise, the readiness probe doesn't allow the pod to run

### CODE REVIEW

**Comment by aojea** [comment]
I'm starting to suspect that we are facing different issues with IPv6
```
Aug 22 11:58:22.746: INFO: kindnet-965h5 started at 2019-08-22 11:44:58 +0000 UTC (0+1 container statuses recorded)
Aug 22 11:58:22.746: INFO: 	Container kindnet-cni ready: true, restart count 0
Aug 22 11:58:22.746: INFO: coredns-d4b9d4d8b-n6884 started at 2019-08-22 11:45:14 +0000 UTC (0+1 container statuses recorded)
Aug 22 11:58:22.746: INFO: 	Container coredns ready: false, restart count 0
```

in this case CoreDNS 1.5.0 is not `ready`, however I added the config 🤷‍♂

**Comment by aojea** [comment]
ok, one issue solved, `CoreDNS-1.5.0` doesn't work after reloading the configuration, you have to recreate the pods and it works. The tests that are failing now is because of the test not picking the `framework.TestContext.IPFamily == ipv6` variable

@neolit123 the changelog of CoreDNS-1.5.2 mentions issues with `ready` and `reload`, is any specific reason to point to 1.5.0 instead of 1.5.2?

**Comment by BenTheElder** [comment]
We added the restart in https://github.com/kubernetes-sigs/kind/pull/803, which also adds the `ready` plugin by way of not patching it out 😅

File: hack/ci/e2e.sh [hunk 1]
```diff
@@ -144,6 +145,7 @@ metadata:
   namespace: kube-system
 ---
 EOF
+    kubectl -n kube-system delete pods -l k8s-app=kube-dns
```
Comment (BenTheElder) [inline hack/ci/e2e.sh]: I stole this line for https://github.com/kubernetes-sigs/kind/pull/803 :^)

**Comment by BenTheElder** [comment]
thanks for this, assuming we can close now 😅

**Comment by aojea** [comment]
fixed by #803

**Pull request closed without merging**